demo/app/android/app/src/main/kotlin/walletsdk/flutter/app/MainActivity.kt

```diff
@@ -821,9 +820,13 @@ class MainActivity : FlutterActivity() {
             val matchedReq = openID4VP.getMatchedSubmissionRequirements(
                 convertToVerifiableCredentialsArray(storedCredentials)
             )
-            return convertVerifiableCredentialsArray(
+            var resp =  convertVerifiableCredentialsArray(
                 matchedReq.atIndex(0).descriptorAtIndex(0).matchedVCs
             )
+            if (resp.isEmpty()) {
+                var typeConstraint = matchedReq.atIndex(0).descriptorAtIndex(0).typeConstraint()
+                result.error("NATIVE_ERR",  "No credentials of type $typeConstraint were found",  "Required credential $typeConstraint is missing from the wallet");
```

Comment:
This is really minor - but I'm wondering why there are two slightly different versions of the same error message here?

Reply:
Both messages seem to have the same information, just worded slightly differently. I would recommend omitting the `details` in such a case.

Reply:
one is message other is details since we read both values in the ui so I rephrased to maintain msg n details consistency